editor/debugger/editor_debugger_node.cpp

```diff
@@ -332,17 +326,19 @@ void EditorDebuggerNode::_notification(int p_what) {
 			if (tabs->get_tab_count() > 1) {
 				tabs->add_theme_style_override(SceneStringName(panel), EditorNode::get_singleton()->get_editor_theme()->get_stylebox(SNAME("DebuggerPanel"), EditorStringName(EditorStyles)));
 			}
-
-			Ref<StyleBox> bottom_panel_margins = EditorNode::get_singleton()->get_editor_theme()->get_stylebox(SNAME("BottomPanel"), EditorStringName(EditorStyles));
-			add_theme_constant_override("margin_top", -bottom_panel_margins->get_margin(SIDE_TOP));
-			add_theme_constant_override("margin_left", -bottom_panel_margins->get_margin(SIDE_LEFT));
-			add_theme_constant_override("margin_right", -bottom_panel_margins->get_margin(SIDE_RIGHT));
-			add_theme_constant_override("margin_bottom", -bottom_panel_margins->get_margin(SIDE_BOTTOM));
+			_update_margins();
 
 			remote_scene_tree->update_icon_max_width();
 		} break;
 
 		case NOTIFICATION_READY: {
+			// TODO: Replace this hack once EditorDebuggerNode is converted to a dock. It should be in the constructor.
```

Comment:
I think this also needs to be done with the new FindInFiles tabs, as it also looks broken now.